managed/CounterStrikeSharp.API/Modules/Utils/EnumUtils.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CounterStrikeSharp.API.Modules.Utils
+{
+    public static class EnumUtils
+    {
+        public static string GetEnumMemberAttributeValue<T>(T enumValue)
```

Comment:
should be nullable if it can return null?

Reply:
String is a referenced type and thus can be already null. Good practice to have checks anyways before using them further. But yes, I have changed it to nullable string.